tmol/score/disulfide/disulfide_energy_term.py

```diff
@@ -0,0 +1,84 @@
+import torch
+
+from ..energy_term import EnergyTerm
+
+from tmol.database import ParameterDatabase
+from tmol.score.disulfide.params import DisulfideGlobalParams
+from tmol.score.disulfide.disulfide_whole_pose_module import (
+    DisulfideWholePoseScoringModule
+)
+
+from tmol.chemical.restypes import RefinedResidueType
+from tmol.pose.packed_block_types import PackedBlockTypes
+from tmol.pose.pose_stack import PoseStack
+
+
+class DisulfideEnergyTerm(EnergyTerm):
+    device: torch.device  # = attr.ib()
+
+    def __init__(self, param_db: ParameterDatabase, device: torch.device):
+        super(DisulfideEnergyTerm, self).__init__(param_db=param_db, device=device)
+
+        self.global_params = DisulfideGlobalParams.from_database(
+            param_db.scoring.disulfide, device
+        )
+        self.device = device
+
+    @classmethod
+    def score_types(cls):
+        import tmol.score.terms.disulfide_creator
+
+        return tmol.score.terms.disulfide_creator.DisulfideTermCreator.score_types()
+
+    def n_bodies(self):
+        return 1
+
+    def setup_block_type(self, block_type: RefinedResidueType):
+        super(DisulfideEnergyTerm, self).setup_block_type(block_type)
+
```

Comment:
Not currently doing anything here, should probably fill the block types with data on the CPU side here.